src/main/java/it/gov/pagopa/pu/workflow/wf/pagopa/send/SendNotificationWFClient.java

```diff
@@ -38,10 +42,11 @@ public String sendNotificationDateRetrieve(String sendNotificationId) {
     String taskQueue = SendNotificationDateRetrieveWFImpl.TASK_QUEUE_SEND_NOTIFICATION_DATE_RETRIEVE;
     String workflowId = generateWorkflowId(sendNotificationId, taskQueue);
 
-    SendNotificationDateRetrieveWF workflow = workflowService.buildWorkflowStub(
+    SendNotificationDateRetrieveWF workflow = workflowService.buildWorkflowStubScheduled(
```

Comment:
changing the already existent method, which actually is used by the Controller, you are forcing the scheduled behavior also for the API!
You have to create a new method instead

ps: can we rename the methods on this class:
* sendNotificationProcess -> startSendNotificationProcess
* sendNotificationDateRetrieve -> startSendNotificationDateRetrieve

while the new methods will be
scheduleSendNotificationDateRetrieve
the nextschedule should be an input, the activity will calculate it

ps: I would not set a fixed date, otherwise we will call SEND for each DP at the same moment!
it's better a random-like behavior, thus let's just schedule using a Duration of 1 hour (if you see, there is an overloaded method of the workflowService which accept a Duration object)